assets/scss/application.scss

```diff
@@ -4,7 +4,7 @@ $path: '/assets/images/';
 $moj-page-width: 1170px;
 $govuk-page-width: $moj-page-width;
 
-@import 'govuk/all';
+@import 'govuk/index';
```

Comment:
It can be helpful to link directly to the release notes https://github.com/alphagov/govuk-frontend/releases/tag/v5.4.0 just to keep everything together and make it super simple for the reviewer.

Reply:
Did this work for you locally? I got an error on this line where index couldn't be found. When I changed it to the following (based on the release notes) it worked:

```
@import "govuk/base";
@import "govuk/core/index";
@import "govuk/objects/index";
@import "govuk/components/index";
@import "govuk/utilities/index";
@import "govuk/overrides/index";
```

Reply:
it did work for me locally, but let me try again.

Reply:
OK now it's not. I thought I had deleted the stylesheets previously to reset the thing but I guess not. I will update this with your suggestion (which also works locally for me).